src/ansys/additive/core/parametric_study/parametric_utils.py

```diff
@@ -39,26 +43,24 @@ def build_rate(
         Laser scan speed.
     layer_thickness : float
         Powder deposit layer thickness.
-    hatch_spacing : float, default: None
+    hatch_spacing : float
         Distance between hatch scan lines.
 
     Returns
     -------
     float
-        Volumetric build rate is returned if hatch spacing is provided.
-        Otherwise, an area build rate is returned. If input units are m/s and m,
-        the output units are m^3/s or m^2/s.
+        The volumetric build rate is returned. For single bead simulations,
+        the default hatch spacing provided in machine constants is used.
+        If input units are m/s and m, the output units are m^3/s or m^2/s.
```

Comment:
```suggestion
        The volumetric build rate is returned. If input units are m/s and m, the output units are m^3/s.
```
This function doesn't take any action based on simulation type. We shouldn't mention single bead simulations here.